src/rest-server/src/models/v2/task.js

```diff
@@ -79,7 +79,7 @@ const get = async (frameworkName, jobAttemptIndex, taskRoleName, taskIndex) => {
   );
   if (taskRoleStatus) {
     taskStatus = taskRoleStatus.taskStatuses.find(
-      (taskRoleStatus) => taskRoleStatus.index === taskIndex,
+      (taskAttemptStatus) => taskAttemptStatus.index === taskIndex,
```

Comment:
Thx for the correction. This should be named `taskStatus`

Reply:
Fixed